sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_decode.py

```diff
@@ -36,130 +47,105 @@
 c_double = struct.Struct('>d')
 
 
-def _decode_null(buffer):
-    # type: (memoryview) -> Tuple[memoryview, None]
+def _decode_null(buffer: memoryview) -> Tuple[memoryview, None]:
     return buffer, None
 
 
-def _decode_true(buffer):
-    # type: (memoryview) -> Tuple[memoryview, bool]
+def _decode_true(buffer: memoryview) -> Tuple[memoryview, Literal[True]]:
     return buffer, True
 
 
-def _decode_false(buffer):
-    # type: (memoryview) -> Tuple[memoryview, bool]
+def _decode_false(buffer: memoryview) -> Tuple[memoryview, Literal[False]]:
     return buffer, False
 
 
-def _decode_zero(buffer):
-    # type: (memoryview) -> Tuple[memoryview, int]
+def _decode_zero(buffer: memoryview) -> Tuple[memoryview, Literal[0]]:
     return buffer, 0
 
 
-def _decode_empty(buffer):
-    # type: (memoryview) -> Tuple[memoryview, List[None]]
+def _decode_empty(buffer: memoryview) -> Tuple[memoryview, List[Any]]:
     return buffer, []
 
 
-def _decode_boolean(buffer):
-    # type: (memoryview) -> Tuple[memoryview, bool]
+def _decode_boolean(buffer: memoryview) -> Tuple[memoryview, bool]:
```

Comment:
had used `Buffer` from `typing_extensions` from Anna's PR but it complained about buffer not having `__get__`